cookiecutter/react_template.yaml

```diff
@@ -1,3 +1,3 @@
 default_context:
```

Comment:
@oudeismetis I changed this manually for now since nobody has touched the gql in a while AND we do not have a way of differentiating I think this is ok?

Reply:
so...this PR broke graphql?

Reply:
It's likely it did,but I would like to fix that separately.

Reply:
We have an upcoming project to start in March I'd like to have this ready by then if possible. 
Don't think I've seen a gql project started since Basta.

Reply:
OK I saw your other PR with a fix for this issue. Up to you if that fix needs to be included here or merged separately.